### ISSUE TITLE
Warnings when creating zip file

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-gui#56]
The zip file is created on the develop branch on the following commit:
```
commit fc3be71d73cb8e0c56633ae88684bbe2e34a7184
Merge: 8717a69 9dec49a
Author: pamasse <pa.masse@outlook.fr>
Date:   Fri May 17 16:02:01 2019 +0200

    Merge pull request #47 from pamasse/fix-empty-ns-default-test
    
    Fix sending empty ns when the advanced options are not open
```

Running `npm install` creates warnings. Can they be ignored? Can warnings always be ignored?
```
$ npm install
npm WARN @angular/compiler-cli@5.2.5 requires a peer of typescript@>=2.4.2 <2.7 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.0.0 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN bootstrap@4.0.0 requires a peer of popper.js@^1.12.9 but none is installed. You must install peer dependencies yourself.
npm WARN tsickle@0.26.0 requires a peer of typescript@>=2.4.2 <2.7 but none is installed. You must install peer dependencies yourself.
npm WARN zonemaster-gui@3.0.1 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

updated 1 package and audited 10861 packages in 18.887s
found 293 vulnerabilities (29 low, 178 moderate, 84 high, 2 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```
Running `npm run release` also creates some warnings. Can they be ignored?
```
$ npm run release

> zonemaster-gui@3.0.1 release /home/matsd/git/zonemaster-gui
> npm run build && node scripts/create_release.js

npm WARN invalid config loglevel="notice"

> zonemaster-gui@3.0.1 build /home/matsd/git/zonemaster-gui
> node scripts/faq.js && ng build -prod

{ [Error: ENOENT: no such file or directory, open './docs/FAQ/gui-faq-ru.md']
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './docs/FAQ/gui-faq-ru.md' }

@angular/compiler-cli@5.2.5 requires typescript@'>=2.4.2 <2.7.0' but 2.7.1 was found instead.
Using this version can result in undefined behaviour and difficult to debug problems.

Please run the following command to install a compatible version of TypeScript.

    npm install typescript@'>=2.4.2 <2.7.0'

To disable this warning run "ng set warnings.typescriptMismatch=false".

(...)
```
After running `npm install typescript@'>=2.4.2 <2.7.0'` and then re-running `npm run release` the typescript warning disappeared.

**Comment by matsduf** [comment]
Is this a bug or not? I guess it depend on the severity of the warning.

**Comment by pamasse** [comment]
Hi Mats, 

All the warning can be ignored. Nodejs as a large ecosystem that evolves quickly. I check regularly these messages.

**Comment by pamasse** [comment]
@pamasse Don't exclude, just delete the json file.

**Comment by pamasse** [comment]
@pamasse Update dependencies (angular, bootstrap) 
Add a small message about warning in develop part of the readme to ignore warning
Add link to the security page of Boostrap and Angular.

**Comment by pamasse** [comment]
> @pamasse Update dependencies (angular, bootstrap)
> Add a small message about warning in develop part of the readme to ignore warning
> Add link to the security page of Boostrap and Angular.

I only updated the dependencies and put a little text about our handling of security issues on them.
I think that's enough.

**Comment by matsduf** [comment]
Will be resolved by #76

**Comment by matsduf** [comment]
Resolved by #76.
